float32 playRate
string bagName
string savePath
bool recordIMU
bool recordStereo
bool recordRgbd
bool recordLidar
bool recordPixhawkIMU
bool recordPx4flow
bool recordVicon
bool recordRtk

